### ISSUE TITLE
Robot/ChronJ ob que 

### ISSUE THREAD

**Issue by joseAbarcaSaavedra** [RancaguaInnova/movil-app#59]
Es posible crear una especie de robot que recorra la página/app revisando que todos los landing estén funcionando, que no tengamos sitios “caidos” ? para que no pase lo de MIHC y lo de los parquimetros  (que no estaban llegando la info de gps)
Y se postee cada cierto tiempo (cada 2 horas por ejemplo cuando no hay problemas) pero si hay problemas con algo postee y etiquete ?

**Comment by jjgumucio** [comment]
Sí, tendría que ser algo tipo _Selenium_ o similar que obtenga alguna etiqueta html determinada.
Si no está la etiqueta/id o lo que sea, que mande una alerta.
